test_tree_coordinates.py

```diff
@@ -0,0 +1,21 @@
+from coordinates import Coordinates
+
+# basic_science
+# |
+# V
+tree1 = Coordinates("tree1", (12.96620, 77.71203))
+tree2 = Coordinates("tree2", (12.96612, 77.71179))
+tree3 = Coordinates("tree3", (12.96603, 77.71156))
+# ganesha_statue
+# |
+# V
+tree4 = Coordinates("tree4", (12.96608, 77.71141))
+tree5 = Coordinates("tree5", (12.96643, 77.71129))
+tree6 = Coordinates("tree6", (12.96682, 77.71115))
+# hostel
+# |
+# V
+tree7 = Coordinates("tree7", (12.96698, 77.71121))
+tree8 = Coordinates("tree8", (12.96698, 77.71121))
+tree9 = Coordinates("tree9", (12.96710, 77.71171))
+# volleyball_court
```

Comment:
added tree coordinates between nodes basic_science, ganesha_statue, hostel and volleyball_court